sdk/basyx/aas/adapter/aasx.py

```diff
@@ -45,7 +45,7 @@
 RELATIONSHIP_TYPE_AAS_SUPL = "http://admin-shell.io/aasx/relationships/aas-suppl"
 
 
-class AASXReader:
+class AASXReader():
```

Comment:
```suggestion
class AASXReader:
```

Brackets are now unnecessary